src/Message.php

```diff
@@ -314,6 +314,10 @@ public function setSender($emailOrAddress, $name = null)
      */
     public function getSender()
     {
+        $headers = $this->getHeaders();
```

Comment:
This change seems unrelated: what call did cause the header to be added? You are only fixing the symptom here, IMO